### ISSUE TITLE
Add ability to POST data to a URL

### ISSUE THREAD

**Issue by Spunkie** [untamed0/Untamed-Now-Playing#27]
Would it be possible to add the ability to POST/PUT the now playing data to a custom URL instead of saving to disk?

**Comment by untamed0** [comment]
Yep, the basic functionality is already there with the Nightbot API intergration, I'll look into adding a custom URL for a future version (probably won't be 0.6, likely 0.7)
